src/main/scala/dotty/dokka/ScalaSignatureProvider.scala

```diff
@@ -27,8 +27,12 @@ class ScalaSignatureProvider(contentConverter: CommentsToContentConverter, logge
             List(extensionSignature(extension)).asJava
         case method: DFunction =>
             List(methodSignature(method)).asJava
+        case enumEntry: DClass if enumEntry.get(IsEnumEntry) != null => 
+            List(enumEntrySignature(enumEntry)).asJava
```

Comment:
imho we can create a helper `JList()` instead of using `List(...).asJava`

Reply:
i've created objects in utils.scala